### ISSUE TITLE
Dont work with Xcode 11 beta 5 and Carthage

### ISSUE THREAD

**Issue by catluc** [scinfu/SwiftSoup#120]
The carthage `update --platform ios` process in Carthage ver. **0.33.0** gets stucked with **SwiftSoup 2.2.0**. I dont know if this issue is SwiftSoup side or Carthage. It will be nice if you can provide some feedback about this issue?

**Comment by funkenstrahlen** [comment]
This is very likely a duplicate of #116

**Comment by catluc** [comment]
> This is very likely a duplicate of #116

Ok thanks, i just reported this to Apple, hope they fix their stuff

**Comment by catluc** [comment]
> This is very likely a duplicate of #116

One question: ¿Do you know if is possible to generate the framework in  Xcode11 beta for SwiftSoup manually?, I tried to do it manually but it only generates the architectures for simulators or for devices but never for both toghether

**Comment by Dschee** [comment]
Hey everyone, while it's not a real fix, as noted by @Igor-Palaguta in #116 a workaround is to set the `SWIFT_OPTIMIZATION_LEVEL` to `-Onone` for release builds. I've done exactly that in my fork, feel free to use it's `master` branch instead of this original repo when installing via Carthage or Accio. Unfortunately, it seems the will not merge it, as can be seen by my long term open PR https://github.com/scinfu/SwiftSoup/pull/117.

In Accio, use this entry instead:

```swift
.package(url: "https://github.com/Dschee/SwiftSoup.git", .branch("master")),
```

In Carthage, use that:

```OGDL
github "Dschee/SwiftSoup" "master"
```

**Comment by catluc** [comment]
> Hey everyone, while it's not a real fix, as noted by @Igor-Palaguta in #116 a workaround is to set the `SWIFT_OPTIMIZATION_LEVEL` to `-Onone` for release builds. I've done exactly that in my fork, feel free to use it's `master` branch instead of this original repo when installing via Carthage or Accio. Unfortunately, it seems the will not merge it, as can be seen by my long term open PR #117.
> 
> In Accio, use this entry instead:
> 
> ```swift
> .package(url: "https://github.com/Dschee/SwiftSoup.git", .branch("master")),
> ```
> 
> In Carthage, use that:
> 
> ```
> github "Dschee/SwiftSoup" "master"
> ```

Did u try this out in the last Xcode 11 beta 6? and what are the implications to set Swift Optimization Level to -0none?

**Comment by Dschee** [comment]
Yes, I did, it's working. Well, it runs slower during runtime. I'm not aware of other implications.

**Comment by catluc** [comment]
> Yes, I did, it's working and is still needed.
> As for `-Onone`, well, it runs slower during runtime. I'm not aware of other implications.

Thanks, but i just checked this out, and it´s still getting stucked on Carthage on same process and same error...... with Xcode 11 beta 6
### Error:
`SwiftSoup.build/Release-iphoneos/SwiftSoup-iOS.build/Objects-normal/armv7/ParseError.bc`

**Comment by Dschee** [comment]
Well, it's working for me though ... weird. I'm running Xcode 11 beta 6 on Mojave, maybe that's it?

**Comment by kylebrowning** [comment]
Stuck for me, Xcode 11 Beta 7

**Comment by rubenvde** [comment]
It's still stuck on the Xcode 11 GM :cold_sweat:

**Comment by scinfu** [comment]
I merged @Dschee PR with SWIFT_OPTIMIZATION_LEVEL to -Onone.
Can someone confirm is fixed? I do not use Carthage.
Thank you
